docs/index.asciidoc

```diff
@@ -2,7 +2,7 @@
 :doctype: book
 :ecs: ECS
 
-include::{asciidoc-dir}/../../shared/versions/stack/7.x.asciidoc[]
+include::{asciidoc-dir}/../../shared/versions/stack/current.asciidoc[]
```

Comment:
Not 100% sure what the effects of this are. Could you expand on this?

Reply:
Saw your comment on this, but I'm just wondering if this will mean past versions of ECS will all link to the latest version of the stack

Reply:
If we back port, yes. But does it matter?

Reply:
Ok I'll trust you on this. I think it'll be fine.

By the way, looking at the other files in this directory, it looks like the variable `ecs_version` is incorrect. It should be 1.2 in `7.4.asciidoc` and in `7.x.asciidoc`. I guess next week after the ECS release we'll increase 7.x to 1.3.